### DISCUSSION TITLE
Sign in  sign up?

### DISCUSSION THREAD

**Discussion by HFolkertH** [cincheo/daquota#33] [Q&A]
Hi Renaud,

how can i sign up initially.? Sign in gives an unknown user error.

**Comment by renaudpawlak** [comment]
Sign up for external users is still work in progress. Do you need an account on platform.dlite.io or on your local installation?
Having an account gives you access to localstorage sync. Is that what you are trying to achieve?
I could create a test account right now if your goal is just to test localstorage sync. But that would just be for testing purpose at that point.

**Reply by HFolkertH** [reply]
Hello Renauld

I just want to test if my 7Gui trials can also be used on my wives Windows laptop and to be started from my own small family website (for sharing some holiday pictures with my family). I have made my own website with Mobirize.com (free)  and host it at a small, cheap Dutch hosting provider.
